File: isogram.js
function is_isogram(word){
	if (typeof(word) === "string")
		throw "Argument should be a string";
	else if(word === ' ')
		return word, false;
	else
		var word = word.toLower();
		for (var i = word.length; i >= 0; i--) {
			if(word[i] === 
		}

File: isogram.py
def is_isogram(word):
    if type(word) != str:
        raise TypeError('Argument should be a string')
    elif word == ' ':
        return word, False
    else:
        word = word.lower()
        for char in word:
            if word.count(char) > 1:
                return word, False
            else:
                return word, True

print is_isogram("Apparently")

